runtime/src/bpftime_prog.cpp

```diff
@@ -26,7 +27,18 @@ bpftime_prog::bpftime_prog(const struct ebpf_inst *insn, size_t insn_cnt,
 {
 	SPDLOG_DEBUG("Creating bpftime_prog with name {}", name);
 	insns.assign(insn, insn + insn_cnt);
-	vm = ebpf_create();
+	const char *vm_name = bpftime::bpftime_get_agent_config().get_vm_name();
+	std::string vm_name_str = (std::string)vm_name;
+
+	if (vm_name_str == "llvm") {
+		SPDLOG_DEBUG("Creating vm with name {}", vm_name_str);
+	} else if (vm_name_str == "ubpf") {
+		SPDLOG_DEBUG("Creating vm with name {}", vm_name_str);
+	} else {
+		SPDLOG_ERROR("Invalid vm name: {}", vm_name_str);
```

Comment:
There shouldn't be an error log. What if a user added his own VM implementation?